src/main/webapp/app/exercises/text/assess/text-submission-assessment.component.ts

```diff
@@ -227,6 +238,88 @@ export class TextSubmissionAssessmentComponent extends TextAssessmentBaseCompone
         this.isAssessor = result?.assessor?.id === this.userId;
     }
 
+    /**
+     * Adds a TextBlockRef, adjusting existing automatic text blocks to fit around the new text block if necessary (and possible).
+     * Example: There already are 2 text blocks:
+     *          - block 1 from index 0 to 10 (automatically generated)
+     *          - block 2 from index 10 to 20 (automatically generated)
+     *          Now, we add a new text block ref with feedback from index 5 to 15.
+     *          Then, we have three text blocks: 0-5, 5-15, 15-20.
+     * If the split conflicts with a manual feedback, we don't add the TextBlockRef at all.
+     *
+     * @param refToAdd The TextBlockRef to add (text block + feedback on it)
+     * @private
```

Comment:
https://typedoc.org/tags/private/
"This tag should generally not be used and may be removed in a future release. The @private tag overrides the visibility of a reflection to be private."
-> has no effect since the method is already private.
```suggestion
```